lib/data/status-codes.js

```diff
@@ -0,0 +1,66 @@
+/* A dump of current https://nodejs.org/docs/latest/api/http.html#http_http_status_codes to avoid including native node modules in browser usage */
```

Comment:
One alternative would be to look for a npm module which offers this data 

https://www.npmjs.com/package/http-status-codes


https://bundlephobia.com/result?p=http-status-codes@1.4.0

Source:

https://github.com/prettymuchbryce/http-status-codes/blob/master/index.js